idp/usso: Allow more than one simultaneous usso IDP

mhilton opened the pull request:
Make the name and description configurable to allow for many usso IDPs
to be configured at the same time.

jujugui commented:
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/candid/69/

mhilton commented:
:shipit:

jujugui commented:
Status: merge request accepted. Url: http://ci-gce.theblues.io:8080/job/candid-merge

mhilton committed:
idp/usso: Allow more than one simultaneous usso IDP

Make the name and description configurable to allow for many usso IDPs
to be configured at the same time.
